configs/fp16/metafile.yml

```diff
@@ -8,9 +8,16 @@ Collections:
       Architecture:
         - Hard Voxelization
     Paper:
-      - https://www.mdpi.com/1424-8220/18/10/3337
-      - https://arxiv.org/abs/1812.05784
+      URL:
+        - https://www.mdpi.com/1424-8220/18/10/3337
+        - https://arxiv.org/abs/1812.05784
+      Title:
+        - 'SECOND: Sparsely Embedded Convolutional Detection'
+        - 'PointPillars: Fast Encoders for Object Detection from Point Clouds'
     README: configs/fp16/README.md
+    Code:
+      URL: https://github.com/open-mmlab/mmcv/blob/c47c9196d067a0900b7b8987a8e82768edab2fff/mmcv/runner/fp16_utils.py#L55
+      Version: v0.1.0
```

Comment:
fp16 should not be 0.1.0? BTW, if the earliest release has the model, we should use v0.1.0 or v0.5.0? (v0.5.0 is the first public release)